pkgs/by-name/si/simulide/clang-fix-protected-field.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/mcusim/mcuinterface.h b/src/mcusim/mcuinterface.h
```

Comment:
Might be worth a comment explaining why this patch exists?

Reply:
Added a comment